PoseEstimation-TFLiteSwift/OpenPose/OpenPosePoseEstimator.swift

```diff
@@ -61,7 +61,9 @@ class OpenPosePoseEstimator: PoseEstimator {
     }
     
     var pairNames: [String]? {
-        return nil
+        return Output.BodyPart.lines.map {
+            return "\($0.from.shortName())-\($0.to.shortName())"
```

Comment:
"\($0.from.shortName())-\($0.to.shortName())" is always String.
This expression never become Optional.

So, I recommend to use [String] for pairNames's type instead [String?].

Reply:
@syjdev 
`pairNames` property is defined on `PoseEstimator` protocol as get-only.
There are 4 classes confirmed `PoseEstimator` protocol:
- `PoseNetPoseEstimator`
- `PEFMCPMPoseEstimator`
- `PEFMHourglassPoseEstimator`
- `OpenPosePoseEstimator` 

But `OpenPosePoseEstimator` return non-nil array on `pairNames` and others return `nil`, because `PoseNetPoseEstimator`, `PEFMCPMPoseEstimator`, and `PEFMHourglassPoseEstimator` don't support multi pose estimation.